src/OrchardCore/OrchardCore.ResourceManagement.Core/ResourceOptions.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using OrchardCore.Settings;
+
+namespace OrchardCore.ResourceManagement.Core;
```

Comment:
Please use `namespace` block

Reply:
not sure why. Is there a guide that we should be using namespace block? namespace blocks only add indentation starting with .NET 6. Anyhow, I changed it per your request.